src/test/ui/impl-trait/explicit-generic-args-for-impl.rs

```diff
@@ -0,0 +1,7 @@
+#![feature(explicit_generic_args_with_impl_trait)]
```

Comment:
Can we put these tests in a directory? `src/test/ui/impl-trait/explicit-generic-args-with-impl-trait`

Reply:
What name should it called when it's moved into a directory? I am not at good at naming things :D

Reply:
Test names can be the same. Just allows us to group them better.